src/lib/layout/shell.svelte

```diff
@@ -112,6 +112,9 @@
     let state: undefined | 'open' | 'closed' | 'icons' = 'closed';
     $: state = $isSidebarOpen ? 'open' : 'closed';
 
+    let isProjectPage;
+    $: isProjectPage = $page.route.id.includes('project-');
```

Comment:
```suggestion
    $: isProjectPage = $page.route?.id.includes('project-');
```
its nullable type so just in case.